_src/_posts/2017-01-25-smarter-mobile-fraud-detection-for-click-spamming.markdown

```diff
@@ -104,12 +105,11 @@ The methods here exposed are a first iteration for fraud detection and classific
 
 We find that this algorithm is strong and flexible to account for differences among applications, where there are significant time differences  between TimeDeltas. The evaluation measures this difference by automatically fitting the best distributions and usesworks with uses it to provide a robust and thorough fraud detection system.
```

Comment:
"time differences (extra space)between". These could be found by searching "  " in the whole document.

"usesworks with uses it " : shouldn't be "which uses it"? Anyway, "**uses**work which **uses** it" looks cacophonic.